website/docs/r/cloud_project_kube.html.markdown

```diff
@@ -207,13 +207,13 @@ The following arguments are supported:
       * `tcp_timeout` - (Optional) Timeout value used for idle IPVS TCP sessions in [RFC3339](https://www.rfc-editor.org/rfc/rfc3339) duration (e.g. `PT60S`). The default value is `PT0S`, which preserves the current timeout value on the system.
       * `tcp_fin_timeout` - (Optional) Timeout value used for IPVS TCP sessions after receiving a FIN in RFC3339 duration (e.g. `PT60S`). The default value is `PT0S`, which preserves the current timeout value on the system.
       * `udp_timeout` - (Optional) timeout value used for IPVS UDP packets in [RFC3339](https://www.rfc-editor.org/rfc/rfc3339) duration (e.g. `PT60S`). The default value is `PT0S`, which preserves the current timeout value on the system.
-* `private_network_id` - (Optional) OpenStack private network (or vRack) ID to use. **Changing this value recreates the resource, including ETCD user data.** Defaults - not use private network.
+* `private_network_id` - (Optional) Private network ID to use. **Changing this value recreates the resource, including ETCD user data.** Defaults - not use private network.
 
 ~> __WARNING__ Updating the private network ID resets the cluster so that all user data is deleted.
 
+* `nodes_subnet_id` - (Optional) Subnet ID to use for nodes. Default uses the first subnet belonging to the private network with id `private_network_id`. This attribute requires `private_network_id` to be defined. **Cannot be updated, it can only be used at cluster creation or reset.**
```

Comment:
```suggestion
* `nodes_subnet_id` - (Optional) Subnet ID to use for nodes, this subnet must belong to private_network_id. Default uses the first subnet belonging to the private network with id `private_network_id`. This attribute requires `private_network_id` to be defined. **Cannot be updated, it can only be used at cluster creation or reset.**
```